promql/promqltest/testdata/operators.test

```diff
@@ -508,3 +508,125 @@ eval instant at 1m 10 atan2 20
 
 eval instant at 1m 10 atan2 NaN
     NaN
+
+clear
+
```

Comment:
```suggestion
// Comparison operators with bool modifier.
```

Reply:
I think it would be better to have a comment here.

Reply:
I've added a comment in 5ef4ae2.